src/main/java/com/angelbroking/smartapi/sample/Examples.java

```diff
@@ -169,6 +169,7 @@ public void getHolding(SmartConnect smartConnect) throws SmartAPIException, IOEx
 	public void getAllHolding(SmartConnect smartConnect) throws SmartAPIException, IOException {
 		// Returns All Holding.
 		JSONObject response = smartConnect.getAllHolding();
+		System.out.println(response);

```

Comment:
System.out.println(response); we should not SOP statements and use logger instead.

Reply:
> System.out.println(response); we should not SOP statements and use logger instead.

remove SOP from all over the project and added logger